mmf/datasets/processors/text_image_processors.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+from math import nan
```

Comment:
Please use
`import math` and then use `math.nan` as it can be confusing where nan came from.